src/queue/queue-data-structure.js

```diff
@@ -5,25 +5,48 @@ class Queue {
   }
 
   canEnqueue() {
-    // ... your code goes here
+    if (this.queueControl.length === this.MAX_SIZE) {
+      return false;
+    }
+    if (this.queueControl.length < this.MAX_SIZE) {
+      return true;
+    }
   }
 
   isEmpty() {
```

Comment:
Si queremos ver si un array está vacío, se puede dejar como: `return !this.stackControl.length` ya que si está vacío el array, el `.length` nos va a retornar 0 (falsy)